packages/backend/src/graphql/schema.graphql

```diff
@@ -12,6 +12,8 @@ type Query {
     first: Int
     "Paginating backwards: The last **n** elements from the page."
     last: Int
+    "Choose ascending or descending order for results."
+    sortOrder: String
```

Comment:
we should make this an enum in the schema, maybe all upper case similar to how we do most enums

Reply:
Ideally, in the future `sortOrder` becomes something more customizable such that you can give it the key you want to sort on e.g. `createdAt` `updatedAt` `activatedAt` but this is good for the time being since we are hardcoding `createdAt` in the pagination code

Reply:
But maybe we can be more specific in the comment "Choose ascending or descending order for results." -> "Ascending or descending order of creation" or something